mr/src/main/java/org/elasticsearch/hadoop/util/EsVersion.java

```diff
@@ -0,0 +1,77 @@
+package org.elasticsearch.hadoop.util;
+
+import org.elasticsearch.hadoop.EsHadoopIllegalArgumentException;
+
+public class EsVersion {
+    public static final EsVersion V_0_X = new EsVersion(0, "0.x");
+    public static final EsVersion V_1_X = new EsVersion(1, "1.x");
+    public static final EsVersion V_2_X = new EsVersion(2, "2.x");
+    public static final EsVersion V_5_X = new EsVersion(3, "3.x");
```

Comment:
I love the addition of this class. One nit: Can we name the `V_5_X` raw name to be `"5.x"` instead of `"3.x"`?